### ISSUE TITLE
[Bug] Cannot run inlets-operator using k3s on Raspberry Pi 4: exec format error

### ISSUE THREAD

**Issue by ameier38** [inlets/inlets-operator#112]
## Expected Behaviour

Using the linux/arm/v6 image in ghcr (specifically `ghcr.io/inlets/inlets-operator:latest@sha256:c20122b628f7d91d0a40b8e88a684391ff800213b775f9fe62a8dabbdf252a94`) should run successfully on Raspberry Pi 4 (arm32v7).

## Current Behaviour

Container crashes with `exec format error`.

## Possible Solution

1. Clone repo.
2. `docker build --build-arg BUILDPLATFORM=linux/arm/v7 --build-arg=TARGETOS=linux --build-arg=TARGETARCH=arm .`

## Steps to Reproduce (for bugs)

1. `helm install inlets inlets/inlets-operator --set image=ghcr.io/inlets/inlets-operator:latest@sha256:c20122b628f7d91d0a40b8e88a684391ff800213b775f9fe62a8dabbdf252a94`

## Context

Trying to run inlets operator on personal Raspberry Pi cluster.

## Your Environment

Raspberry Pi 4 (arm32v7)

* inlets-operator version, find via `kubectl get deploy inlets-operator -o wide`
0.10.1

* Kubernetes distribution i.e. minikube v0.29.0., KinD v0.5.1, Docker Desktop:
k3s

* Kubernetes version `kubectl version`:
```
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.3", GitCommit:"1e11e4a2108024935ecfcb2912226cedeafd99df", GitTreeState:"clean", BuildDate:"2020-10-14T12:50:19Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"windows/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.4+k3s1", GitCommit:"2532c10faad43e2b6e728fdcc01662dc13d37764", GitTreeState:"clean", BuildDate:"2020-11-18T22:10:59Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"linux/arm"}
```

* Operating System and version (e.g. Linux, Windows, MacOS):
Windows

* Cloud provisioner:
DigitalOcean

**Comment by alexellis** [comment]
Fixed in 68fa618

The multiarch build failed. Please use the latest helm chart which has just been updated.

**Comment by ameier38** [comment]
Thanks for the quick fix. I just updated and can confirm it runs successfully.
